src/liquid/components/ld-pagination/ld-pagination.shadow.css

```diff
@@ -5,9 +5,21 @@
   --ld-pagination-column-size: calc(
     var(--ld-pagination-item-size) + var(--ld-pagination-items-space)
   );
+  --ld-pagination-dot-color: var(--ld-thm-primary-alpha-low);
+  --ld-pagination-dots-size: var(--ld-pagination-dots-size-md);
+  --ld-pagination-dots-size-sm: 0.25rem;
+  --ld-pagination-dots-size-md: 0.375rem;
+  --ld-pagination-dots-size-lg: 0.5rem;
   --ld-pagination-more-indicator-font: var(--ld-typo-body-m);
   --ld-pagination-item-size: var(--ld-pagination-md-item-size);
   --ld-pagination-items-space: var(--ld-sp-4);
+  --ld-pagination-dots-space: var(--ld-pagination-dots-space-md);
+  --ld-pagination-dots-space-sm: var(--ld-sp-8);
+  --ld-pagination-dots-space-md: var(--ld-sp-12);
+  --ld-pagination-dots-space-lg: var(--ld-sp-16);
+  --ld-pagination-marker-color: var(--ld-thm-primary-alpha-lowest);
+  --ld-pagination-primary-color: var(--ld-thm-primary);
+  --ld-pagination-selected-color: var(--thm-primary-active);
 
   align-items: center;
   display: inline-flex;
```

Comment:
Please remove the `overflow: hidden;` here. I don't know what purpose it has here, but it cuts the focus outline in dots mode, if implemented as suggested below. If it has a purpose, please check if we can do without it. If not, please comment.